{
    "active_lock_reason": null,
    "assignee": null,
    "assignees": [],
    "author_association": "NONE",
    "body": "Hi,\r\n\r\nI'm trying to extract the topography from a grid along a certain path, going from A to B.\r\nFor this purpose, here is the code:\r\n\r\n```\r\nregion = [14.1, 15.2, 41.0, 41.6]\r\ngrid = pygmt.datasets.load_earth_relief(resolution=\"01s\", region=region)\r\nprofile = '14.33/41.43/14.93/41.14'         # these are the coordinates of A and B\r\n\r\ntrack   = pygmt.grdtrack(points = None, grid = grid,\r\n                                          profile = profile)\r\n```\r\n\r\n This is the error I get. It occurs with any grid provided, as long as the \"profile\" option is used and \"points\" is set to None.\r\n```\r\nFile \"/Users/giovanni_work/anaconda3/envs/pygmt/lib/python3.9/site-packages/pygmt/helpers/utils.py\", line 69, in data_kind\r\n    raise GMTInvalidInput(\"No input data provided.\")\r\n\r\nGMTInvalidInput: No input data provided.\r\n```\r\nI don't know if this is a bug or I'm just wrongly using the profile/points option is grdtrack.\r\nAny clues on how to solve this?\r\n\r\nThank you!\r\n\r\n\r\n**System information**\r\n\r\n```\r\nPyGMT information:\r\n  version: v0.5.0\r\nSystem information:\r\n  python: 3.9.10 | packaged by conda-forge | (main, Feb  1 2022, 21:28:27)  [Clang 11.1.0 ]\r\n  executable: /Users/rex/anaconda3/envs/pygmt/bin/python\r\n  machine: macOS-11.6.1-x86_64-i386-64bit\r\nDependency information:\r\n  numpy: 1.22.3\r\n  pandas: 1.4.1\r\n  xarray: 2022.3.0\r\n  netCDF4: 1.5.8\r\n  packaging: 21.3\r\n  ghostscript: 9.54.0\r\n  gmt: 6.3.0\r\nGMT library information:\r\n  binary dir: /Users/giovanni_work/anaconda3/envs/pygmt/bin\r\n  cores: 16\r\n  grid layout: rows\r\n  library path: /Users/giovanni_work/anaconda3/envs/pygmt/lib/libgmt.dylib\r\n  padding: 2\r\n  plugin dir: /Users/giovanni_work/anaconda3/envs/pygmt/lib/gmt/plugins\r\n  share dir: /Users/giovanni_work/anaconda3/envs/pygmt/share/gmt\r\n  version: 6.3.0\r\n```\r\n",
    "closed_at": "2022-06-06T05:48:24Z",
    "closed_by": {
        "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
        "events_url": "https://api.github.com/users/seisman/events{/privacy}",
        "followers_url": "https://api.github.com/users/seisman/followers",
        "following_url": "https://api.github.com/users/seisman/following{/other_user}",
        "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/seisman",
        "id": 3974108,
        "login": "seisman",
        "node_id": "MDQ6VXNlcjM5NzQxMDg=",
        "organizations_url": "https://api.github.com/users/seisman/orgs",
        "received_events_url": "https://api.github.com/users/seisman/received_events",
        "repos_url": "https://api.github.com/users/seisman/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/seisman"
    },
    "comments": 7,
    "comments_content": [
        {
            "author_association": "NONE",
            "body": "\ud83d\udc4b Thanks for opening your first issue here! Please make sure you filled out the template with as much detail as possible. You might also want to take a look at our [contributing guidelines](https://github.com/GenericMappingTools/pygmt/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/GenericMappingTools/pygmt/blob/main/CODE_OF_CONDUCT.md).\n",
            "created_at": "2022-03-17T14:20:31Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/1827#issuecomment-1070963088",
            "id": 1070963088,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827",
            "node_id": "IC_kwDOBRZfO84_1ZmQ",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1070963088/reactions"
            },
            "updated_at": "2022-03-17T14:20:31Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1070963088",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/in/4141?v=4",
                "events_url": "https://api.github.com/users/welcome%5Bbot%5D/events{/privacy}",
                "followers_url": "https://api.github.com/users/welcome%5Bbot%5D/followers",
                "following_url": "https://api.github.com/users/welcome%5Bbot%5D/following{/other_user}",
                "gists_url": "https://api.github.com/users/welcome%5Bbot%5D/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/apps/welcome",
                "id": 30606887,
                "login": "welcome[bot]",
                "node_id": "MDM6Qm90MzA2MDY4ODc=",
                "organizations_url": "https://api.github.com/users/welcome%5Bbot%5D/orgs",
                "received_events_url": "https://api.github.com/users/welcome%5Bbot%5D/received_events",
                "repos_url": "https://api.github.com/users/welcome%5Bbot%5D/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/welcome%5Bbot%5D/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/welcome%5Bbot%5D/subscriptions",
                "type": "Bot",
                "url": "https://api.github.com/users/welcome%5Bbot%5D"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "Ah yes, I think I know what's happening, this might be a bug introduced in #1189.\r\n\r\nIn the meantime, you can try using this as a workaround. You can adjust the `num=250` to the number of points you want along the line:\r\n\r\n```python\r\nimport pandas as pd\r\nimport pygmt\r\nimport numpy as np\r\n\r\nregion = [14.1, 15.2, 41.0, 41.6]\r\ngrid = pygmt.datasets.load_earth_relief(resolution=\"01s\", region=region)\r\n\r\nprofile = \"14.33/41.43/14.93/41.14\"  # these are the coordinates of A and B\r\n\r\npoints = pd.DataFrame(\r\n    data=np.linspace(start=(14.33, 41.43), stop=(14.93, 41.14), num=250),\r\n    columns=[\"x\", \"y\"],\r\n)\r\n\r\ntrack = pygmt.grdtrack(points=points, grid=grid, newcolname=\"elevation\")\r\nprint(track)\r\n```\r\nproduces\r\n```python\r\n\tx\ty\televation\r\n0\t14.330000\t41.430000\t1294.000000\r\n1\t14.332410\t41.428835\t1389.962382\r\n2\t14.334819\t41.427671\t1445.901806\r\n3\t14.337229\t41.426506\t1397.092802\r\n4\t14.339639\t41.425341\t1472.605211\r\n...\t...\t...\t...\r\n245\t14.920361\t41.144659\t235.882504\r\n246\t14.922771\t41.143494\t215.088483\r\n247\t14.925181\t41.142329\t170.312853\r\n248\t14.927590\t41.141165\t158.030026\r\n249\t14.930000\t41.140000\t173.000000\r\n\r\n250 rows \u00d7 3 columns\r\n```\r\n\r\nbased on this chunk of code: https://github.com/weiji14/pyissm/blob/2a30cfa461b81493f41021bac0c3621a3e30c2ec/plot_figures.py#L105-L141",
            "created_at": "2022-03-17T14:29:59Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/1827#issuecomment-1070967442",
            "id": 1070967442,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827",
            "node_id": "IC_kwDOBRZfO84_1aqS",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1070967442/reactions"
            },
            "updated_at": "2022-03-17T14:29:59Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1070967442",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
                "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
                "followers_url": "https://api.github.com/users/weiji14/followers",
                "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
                "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/weiji14",
                "id": 23487320,
                "login": "weiji14",
                "node_id": "MDQ6VXNlcjIzNDg3MzIw",
                "organizations_url": "https://api.github.com/users/weiji14/orgs",
                "received_events_url": "https://api.github.com/users/weiji14/received_events",
                "repos_url": "https://api.github.com/users/weiji14/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/weiji14"
            }
        },
        {
            "author_association": "NONE",
            "body": "Ok, I see. \r\nFor now I'm going to use the  workaround you proposed. Thanks!",
            "created_at": "2022-03-18T06:31:03Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/1827#issuecomment-1072072462",
            "id": 1072072462,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827",
            "node_id": "IC_kwDOBRZfO84_5ocO",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1072072462/reactions"
            },
            "updated_at": "2022-03-18T06:31:03Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1072072462",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/15686729?v=4",
                "events_url": "https://api.github.com/users/19giovi87/events{/privacy}",
                "followers_url": "https://api.github.com/users/19giovi87/followers",
                "following_url": "https://api.github.com/users/19giovi87/following{/other_user}",
                "gists_url": "https://api.github.com/users/19giovi87/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/19giovi87",
                "id": 15686729,
                "login": "19giovi87",
                "node_id": "MDQ6VXNlcjE1Njg2NzI5",
                "organizations_url": "https://api.github.com/users/19giovi87/orgs",
                "received_events_url": "https://api.github.com/users/19giovi87/received_events",
                "repos_url": "https://api.github.com/users/19giovi87/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/19giovi87/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/19giovi87/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/19giovi87"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "Reopening the issue so that we can better track the bug report and fix it later.",
            "created_at": "2022-03-18T08:40:01Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/1827#issuecomment-1072181105",
            "id": 1072181105,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827",
            "node_id": "IC_kwDOBRZfO84_6C9x",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1072181105/reactions"
            },
            "updated_at": "2022-03-18T08:40:01Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1072181105",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
                "events_url": "https://api.github.com/users/seisman/events{/privacy}",
                "followers_url": "https://api.github.com/users/seisman/followers",
                "following_url": "https://api.github.com/users/seisman/following{/other_user}",
                "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/seisman",
                "id": 3974108,
                "login": "seisman",
                "node_id": "MDQ6VXNlcjM5NzQxMDg=",
                "organizations_url": "https://api.github.com/users/seisman/orgs",
                "received_events_url": "https://api.github.com/users/seisman/received_events",
                "repos_url": "https://api.github.com/users/seisman/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/seisman"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "There are two different ways to specify input track coordinates to `grdtrack`:\r\n\r\n#. Pass a file or array to `points`\r\n#. Set `profile` (grdtrack's `-E` option)\r\n\r\nPR #1867 tries to fix the issue by detecting if `points` or `profile` is given. Because `points` is a required parameter, users have to use \r\n```\r\npygmt.grdtrack(points=None, grid=grid, profile=profile)\r\n```\r\nwhich is not friendly. \r\n\r\nI think we should reorder the function definition to `grdtrack(grid, points=None, **kwargs`) so that `points` is optional. The change should be done in a backward-compatible way and goes into the deprecatation cycle. That means #1867 can't be merged in the v0.6.1 version.\r\n",
            "created_at": "2022-04-06T02:03:11Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/1827#issuecomment-1089661694",
            "id": 1089661694,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827",
            "node_id": "IC_kwDOBRZfO85A8ur-",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1089661694/reactions"
            },
            "updated_at": "2022-04-06T02:03:11Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1089661694",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
                "events_url": "https://api.github.com/users/seisman/events{/privacy}",
                "followers_url": "https://api.github.com/users/seisman/followers",
                "following_url": "https://api.github.com/users/seisman/following{/other_user}",
                "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/seisman",
                "id": 3974108,
                "login": "seisman",
                "node_id": "MDQ6VXNlcjM5NzQxMDg=",
                "organizations_url": "https://api.github.com/users/seisman/orgs",
                "received_events_url": "https://api.github.com/users/seisman/received_events",
                "repos_url": "https://api.github.com/users/seisman/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/seisman"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "Ok, considering that there is a workaround, I guess we could merge #1867 in v0.7.0. But shouldn't we start to raise a `FutureWarning` in v0.6.1 that the `grid` parameter should be first and `points` is second? E.g. use a modified version of the `@check_data_input_order` decorator used in https://github.com/GenericMappingTools/pygmt/issues/1479?\r\n\r\nOr actually, can't we just change the function definition from `def grdtrack(points, grid, ..., **kwargs):` to `def grdtrack(points=None, grid=None, ..., **kwargs):`?",
            "created_at": "2022-04-06T03:01:51Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/1827#issuecomment-1089718088",
            "id": 1089718088,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827",
            "node_id": "IC_kwDOBRZfO85A88dI",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1089718088/reactions"
            },
            "updated_at": "2022-04-06T03:05:13Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1089718088",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
                "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
                "followers_url": "https://api.github.com/users/weiji14/followers",
                "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
                "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/weiji14",
                "id": 23487320,
                "login": "weiji14",
                "node_id": "MDQ6VXNlcjIzNDg3MzIw",
                "organizations_url": "https://api.github.com/users/weiji14/orgs",
                "received_events_url": "https://api.github.com/users/weiji14/received_events",
                "repos_url": "https://api.github.com/users/weiji14/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/weiji14"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "> Or actually, can't we just change the function definition from `def grdtrack(points, grid, ..., **kwargs):` to `def grdtrack(points=None, grid=None, ..., **kwargs):`?\r\n\r\n`grid` is a required parameter for `grdtrack`, and most other grd* functions have a definition like `def grdxxx(grid, ..., **kwargs)`. I think `grdtrack` should also follow the same style.\r\n\r\n> But shouldn't we start to raise a `FutureWarning` in v0.6.1 that the `grid` parameter should be first and `points` is second?\r\n\r\nI'm not sure if I understand you correctly. Do you mean changing the function definition or not? If we change the function definition, then the change should be in v0.7.0, not v0.6.1 following the semantic versioning. If we don't change the function definition, the `FutureWarning` doesn't make any sense.",
            "created_at": "2022-04-06T04:22:18Z",
            "html_url": "https://github.com/GenericMappingTools/pygmt/issues/1827#issuecomment-1089799172",
            "id": 1089799172,
            "issue_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827",
            "node_id": "IC_kwDOBRZfO85A9QQE",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1089799172/reactions"
            },
            "updated_at": "2022-04-06T04:22:18Z",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/comments/1089799172",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/3974108?v=4",
                "events_url": "https://api.github.com/users/seisman/events{/privacy}",
                "followers_url": "https://api.github.com/users/seisman/followers",
                "following_url": "https://api.github.com/users/seisman/following{/other_user}",
                "gists_url": "https://api.github.com/users/seisman/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/seisman",
                "id": 3974108,
                "login": "seisman",
                "node_id": "MDQ6VXNlcjM5NzQxMDg=",
                "organizations_url": "https://api.github.com/users/seisman/orgs",
                "received_events_url": "https://api.github.com/users/seisman/received_events",
                "repos_url": "https://api.github.com/users/seisman/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/seisman/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/seisman/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/seisman"
            }
        }
    ],
    "comments_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827/comments",
    "created_at": "2022-03-17T14:20:19Z",
    "events_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827/events",
    "html_url": "https://github.com/GenericMappingTools/pygmt/issues/1827",
    "id": 1172432495,
    "labels": [
        {
            "color": "ee0701",
            "default": true,
            "description": "Something isn't working",
            "id": 563309827,
            "name": "bug",
            "node_id": "MDU6TGFiZWw1NjMzMDk4Mjc=",
            "url": "https://api.github.com/repos/GenericMappingTools/pygmt/labels/bug"
        }
    ],
    "labels_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827/labels{/name}",
    "locked": false,
    "milestone": {
        "closed_at": "2022-07-05T13:19:49Z",
        "closed_issues": 100,
        "created_at": "2021-11-06T22:12:34Z",
        "creator": {
            "avatar_url": "https://avatars.githubusercontent.com/u/23487320?v=4",
            "events_url": "https://api.github.com/users/weiji14/events{/privacy}",
            "followers_url": "https://api.github.com/users/weiji14/followers",
            "following_url": "https://api.github.com/users/weiji14/following{/other_user}",
            "gists_url": "https://api.github.com/users/weiji14/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/weiji14",
            "id": 23487320,
            "login": "weiji14",
            "node_id": "MDQ6VXNlcjIzNDg3MzIw",
            "organizations_url": "https://api.github.com/users/weiji14/orgs",
            "received_events_url": "https://api.github.com/users/weiji14/received_events",
            "repos_url": "https://api.github.com/users/weiji14/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/weiji14/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/weiji14/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/weiji14"
        },
        "description": null,
        "due_on": null,
        "html_url": "https://github.com/GenericMappingTools/pygmt/milestone/12",
        "id": 7333793,
        "labels_url": "https://api.github.com/repos/GenericMappingTools/pygmt/milestones/12/labels",
        "node_id": "MI_kwDOBRZfO84Ab-eh",
        "number": 12,
        "open_issues": 0,
        "state": "closed",
        "title": "0.7.0",
        "updated_at": "2022-07-05T13:19:49Z",
        "url": "https://api.github.com/repos/GenericMappingTools/pygmt/milestones/12"
    },
    "node_id": "I_kwDOBRZfO85F4eZv",
    "number": 1827,
    "performed_via_github_app": null,
    "reactions": {
        "+1": 0,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 0,
        "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827/reactions"
    },
    "repository_url": "https://api.github.com/repos/GenericMappingTools/pygmt",
    "state": "closed",
    "state_reason": "completed",
    "timeline_url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827/timeline",
    "title": "grdtrack not working with \"profile\" option",
    "updated_at": "2022-06-06T05:48:24Z",
    "url": "https://api.github.com/repos/GenericMappingTools/pygmt/issues/1827",
    "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/15686729?v=4",
        "events_url": "https://api.github.com/users/19giovi87/events{/privacy}",
        "followers_url": "https://api.github.com/users/19giovi87/followers",
        "following_url": "https://api.github.com/users/19giovi87/following{/other_user}",
        "gists_url": "https://api.github.com/users/19giovi87/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/19giovi87",
        "id": 15686729,
        "login": "19giovi87",
        "node_id": "MDQ6VXNlcjE1Njg2NzI5",
        "organizations_url": "https://api.github.com/users/19giovi87/orgs",
        "received_events_url": "https://api.github.com/users/19giovi87/received_events",
        "repos_url": "https://api.github.com/users/19giovi87/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/19giovi87/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/19giovi87/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/19giovi87"
    }
}